src/java/org/apache/cassandra/config/Config.java

```diff
@@ -397,6 +397,8 @@ public static class SSTableConfig
     @Replaces(oldName = "max_mutation_size_in_kb", converter = Converters.KIBIBYTES_DATASTORAGE, deprecated = true)
     public DataStorageSpec.IntKibibytesBound max_mutation_size;
 
+    public boolean bulk_load_enabled = true;
```

Comment:
@Runtian you should introduce and document this property in `cassandra.yml` as well

Reply:
also, I think this variable should be `volatile`.